nuxeo-liveconnect-dropbox-core/src/main/java/org/nuxeo/ecm/liveconnect/dropbox/DropboxBlobProvider.java

```diff
@@ -105,16 +110,18 @@ public InputStream getThumbnail(ManagedBlob blob) throws IOException {
         LiveConnectFileInfo fileInfo = toFileInfo(blob);
         String filePath = fileInfo.getFileId();
         try {
-            DbxClient.Downloader downloader = getDropboxClient(getCredential(fileInfo)).startGetThumbnail(
-                    DbxThumbnailSize.w64h64, DbxThumbnailFormat.bestForFileName(filePath, DbxThumbnailFormat.JPEG),
-                    filePath, null);
-
+            DbxDownloader downloader = getDropboxClient(getCredential(fileInfo)).files()
+                    .getThumbnailBuilder(filePath)
+                    .withFormat(ThumbnailFormat.JPEG)
+                    .withSize(ThumbnailSize.W64H64)
+                    .start();
```

Comment:
It seems there's some format issue, all `.sth` should be aligned with `.files()`.